cli/src/helpers/local/configFileGlobal.ts

```diff
@@ -1,9 +1,9 @@
 import chalk from 'chalk';
 import fs from 'fs';
-import { getTypedStoatConfig, readStoatConfig } from 'stoat-action/src/configHelpers';
-import {StoatConfigSchema} from "stoat-action/src/schemas/stoatConfigSchema";
-import { getTemplate } from 'stoat-action/src/templateHelpers';
-import { Template } from 'stoat-action/src/types';
+import { getTypedStoatConfig, readStoatConfig } from '../../../../action/src/configHelpers';
+import {StoatConfigSchema} from "../../../../action/src/schemas/stoatConfigSchema";
+import { getTemplate } from '../../../../action/src/templateHelpers';
+import { Template } from '../../../../action/src/types';
```

Comment:
I am surprised that this works. Are the dependencies messed up? Like if the `templateHelpers` depends on a library that's only in the `action` project, the `cli` project won't have that, and the code will fail.

Reply:
I think this is a real issue, based on the latest build failure.

Reply:
it looks like the "correct" thing is to install the action first

Reply:
Nice.